.github/workflows/main_ci.yml

```diff
@@ -3,8 +3,6 @@ name: Elixir CI
 on:
```

Comment:
Renaming this file will break the `build` badge in the `README.md` 💔 
Also, I don't understand the _need_ to rename it ... 🤷‍♂️ 
What am I missing? 💭

Reply:
I renamed the pipeline job to go in tandem with the other changes pertaining to readability. I think `Elixir CI` is too broad. Having it changed to `PR CI` is more discernable, as we know the `PR_ci.yml` workflow is running and pertains to the PR, not a merge to the `main` branch (which occurs on another file).

Reply:
Renamed to `pr.yml` clear enough. no need for mixed case in the file name. 👌